### PR TITLE
Add an 'unsupported on android' warning to 2d skeleton documentation

### PR DESCRIPTION
**Pull request by kjav**

### CODE REVIEW

**Comment by kjav** [comment]
Hi @Calinou - good point, I'll add in something about HTML5. However the GLES3 renderer is also broken in a different way; the issue here explains that there can be colour problems with it https://github.com/godotengine/godot/issues/35772#issuecomment-582753896

**Comment by Calinou** [comment]
You can amend your pull request instead of opening a new one :slightly_smiling_face:

**Comment by kjav** [comment]
@Calinou Thanks - I was going to close this one as it seems you covered it in that other issue you opened (https://github.com/godotengine/godot/pull/41927) - unless that one is just in the editor and not in the docs?

**Comment by Calinou** [comment]
@kjav That PR is only for the class reference, not the documentation page.

**Comment by kjav** [comment]
Thanks Calinou, I didn't realise that. I'll update the message

File: tutorials/animation/2d_skeletons.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 2D skeletons
 ============
 
+.. warning::
+
+   Please note that 2D skeletons are currently not supported on Android (see `this issue
+   <https://github.com/godotengine/godot/issues/35772>`_ for more information).
```
Comment (NathanLovato) [inline tutorials/animation/2d_skeletons.rst:9]:
```suggestion
    There are known issues with 2D skeletons on mobile and web platforms with the GLES2 renderer. We
    recommend using the GLES3 renderer if your project relies on Skeleton2D for now.
```

**Comment by NathanLovato** [comment]
I took the liberty to edit the message directly to save you some time.

**Comment by NathanLovato** [comment]
Thanks for your contribution!

**Pull request merged**